Scripts/Python/xIniHelper.py

```diff
@@ -153,40 +153,32 @@ def removeEntry(self,idx):
         except IndexError:
             pass
 
-    def findByCommand(self,cmd,idx=0):
-        idx = 0
-        for entry in self.entries[idx:]:
+    def findByCommand(self, cmd, idx=0):
+        for index, entry in enumerate(self.entries):
             if entry.command == cmd:
-                return entry,idx
-            idx += 1
-        return None,-1
+                return entry, index
+        return None, -1
 
-    def findByFirstValue(self,value,idx=0):
-        idx = 0
-        for entry in self.entries[idx:]:
+    def findByFirstValue(self, value, idx=0):
+        for index, entry in enumerate(self.entries):
             if len(entry.values) > 0 and entry.values[0] == value:
```

Comment:
```suggestion
            if entry.values and entry.values[0] == value:
```

Reply:
Would you mind fixing this suggestion so it can be applied from the GitHub UI? I think you accidentally made the suggestion on the "old" side of the changeset and it won't let me apply it. I could apply it by hand locally and push, but I'm a little lazy. ;)